particles/Tutorial/Kotlin/4_Handles/Handles.arcs

```diff
@@ -0,0 +1,29 @@
+// Define a schema that allows us to store a person's name 
+schema Person
```

Comment:
@piotrswigon @Cypher1 should we be retrofitting the pre-TTT Tutorials to use inline schemas and, perhaps, `alias` for improved syntactic sugar?

Reply:
Yes we should. Also we need to have full clarity on how alias should be used, which is an action item for me...

Reply:
Yes, we should not introduce new warnings in the code. I'd like for our warnings to suggest the necessary changes to easy fixing them but currently it isn't high priority.